src/main/java/com/urswolfer/intellij/plugin/gerrit/git/GerritGitUtil.java

```diff
@@ -114,16 +114,8 @@
     public Optional<GitRepository> getRepositoryForGerritProject(Project project, String gerritProjectName) {
         final Iterable<GitRepository> repositoriesFromRoots = getRepositories(project);
         for (GitRepository repository : repositoriesFromRoots) {
-            for (GitRemote remote : repository.getRemotes()) {
-                if (remote.getName().equals(gerritProjectName)) {
-                    return Optional.of(repository);
-                }
-                for (String remoteUrl : remote.getUrls()) {
-                    remoteUrl = UrlUtils.stripGitExtension(remoteUrl);
-                    if (remoteUrl != null && remoteUrl.endsWith(gerritProjectName)) {
-                        return Optional.of(repository);
-                    }
-                }
+            if (repository.getRoot().getCanonicalPath().equals(project.getBaseDir().getCanonicalPath())) {
```

Comment:
I think this solution looks a bit too simple to be true :)

Are you really sure that the code above is not required anymore?